docs/js/aml_playground_validation.js

```diff
@@ -58476,7 +58476,7 @@ var ViewModel = /** @class */ (function () {
         this.changesFromLastUpdate = 0;
         this.someModelChanged = false;
         this.RELOAD_PERIOD = 1000;
-        this.base = window.location.origin;
+        this.base = window.location.href.toString().replace('/validation.html', '');
```

Comment:
there is no way to make this a bit more generic instead of explicitly removing `validation.html`, remove `*.html` or get that `validation.html` dynamically (is it not part of the `window.location` object?)?

Reply:
I can't think of a generic way right now. This is how it worked before I tried to use `origin`. `validation.html` is not part of `window.location`.